### DISCUSSION TITLE
Sync between GitHub/Docker Hub

### DISCUSSION THREAD

**Discussion by GauthierMenier** [seriousm4x/UpSnap#684] [Feature requests]
Hello guys,
Is it possible to link repo releases to any Docker Hub repository ?
Container Manager (in Synology DSM package) works with Docker Hub.
I think to do so, an account is required in docker hub and you can link any repo to your profile.
Best regards,

**Comment by GauthierMenier** [comment]
https://felixcentmerino.wordpress.com/containers/sync-up-docker-hub-and-github-to-build-images-automatically/

**Comment by GauthierMenier** [comment]
Anyone ??

**Comment by seriousm4x** [comment]
Sorry for the late response. I originally used docker hub before they anncounced their policy change for free tier accounts. After that i switched to ghcr.

Of course it is possible to sync images to docker hub, but does it have any advantages? I'm pretty sure that you can also use ghcr images on Synology.

**Reply by GauthierMenier** [reply]
Unfortunately, issue is known.. :( Container manager is not compatible with GHCR for the moment. We can use docker directly but we loose container manager feature which can tell us if new version is available or not.

**Reply by GauthierMenier** [reply]
We can't link directly ghcr to docker hub if repo is not yours.. only the owner can do that

**Comment by seriousm4x** [comment]
I've added docker hub to the build pipeline. Here is the repo: https://hub.docker.com/repository/docker/seriousm4x/upsnap/

Currently there is only a beta tag, but once the next version releases, it will also be there.

**Reply by GauthierMenier** [reply]
Thanks a lot. It will help a lot of synology users to use this repo easily.
